.github/actions/post-test/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
     description: 'site for datadog'
     required: false
     default: 'us3.datadoghq.com'
+  gcp-credentials:
+    description: 'gcp'
+    required: false
```

Comment:
Why required false? Shouldn't it be there always from wherever post-test is called? Or do we want to skip from some places?

Reply:
we are skipping in some places